xgi/classes/simplicialcomplex.py

```diff
@@ -189,8 +193,8 @@ def add_simplex(self, members, id=None, **attr):
         >>> S = xgi.SimplicialComplex()
         >>> S.add_simplex([1, 2, 3])
         >>> S.edges.members() # doctest: +NORMALIZE_WHITESPACE
-        [frozenset({1, 2, 3}), frozenset({1, 2}),
-            frozenset({1, 3}), frozenset({2, 3})]
+        [frozenset({1, 2, 3}), frozenset({2, 3}),
```

Comment:
From what I've seen, in this case yes. I noticed the order was not reliable with string nodes.. so I tested for set equality in those cases.

Reply:
But Github action might be proving me wrong right now, this order thing is all a bit mysterious still. For strings I noticed it would change only when restarting the python kernel for example.

Reply:
From Github actions it seems order is deterministic in 3.11 but random in 3.7... there are more tests that don't pass in 3.7.

Reply:
Update: not random in 3.7, deterministic. But consistently a different order than in 3.11.

Reply:
Done: checking for set equality now.